contracts/NUSElections.sol

```diff
@@ -214,7 +214,7 @@ contract NUSElections {
         return totalVotes;
     }
 
-    function test(uint a, uint b) public returns(uint) {
+    function test(uint a, uint b) public public returns(uint) {
```

Comment:
think this should be public pure instead of public public?

understand just a test, for compiling purpose